_includes/head.html

```diff
@@ -3,16 +3,16 @@
 {% endif %}
 
 <head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <title>{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}</title>
-  <meta name="description" content="Write an awesome description for your new site here. You can edit this line in _config.yml. It will appear in your document head meta (for Google search resu...">
+	<title>{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}</title>
+	<meta name="description" content="Write an awesome description for your new site here. You can edit this line in _config.yml. It will appear in your document head meta (for Google search resu...">
 
-  <link rel="stylesheet" href="{{site.baseurl}}/assets/main.css">
-  <link rel="stylesheet" href="{{site.baseurl}}/assets/site.css">
-  <link rel="alternate" type="application/rss+xml" title="SMC Style Guide" href="/feed.xml">
-  
-  
+	<link rel="stylesheet" href="{{site.baseurl}}/assets/main.css">
+	<link rel="alternate" type="application/rss+xml" title="SMC Style Guide" href="/feed.xml">
+
+	<script src="https://use.typekit.net/xzs4qjr.js"></script>
+	<script>try{Typekit.load({ async: true });}catch(e){}</script>
 </head>
```

Comment:
Once we establish a rule,  please update your text editor to enforce. And do call out inconsistencies in any PRs.